### DISCUSSION TITLE
type of proxies would it be?

### DISCUSSION THREAD

**Discussion by drakewebs1** [masterking32/MasterHamsterKombatBot#39] [Q&A] [answered]
![pCaptura](https://github.com/user-attachments/assets/73a86eae-37cd-4847-9540-a8f4ba875139)
**Hello, good evening, sorry for the time, what type of proxy should be used, the ones in the image, can you use one of that type, or do you have to be user:password@ip:port? thanks for your understanding**

**Comment by masterking32** [comment, marked as answer]
Hello,
You cannot use a proxy list, but you can assign a proxy for each account. For example:
```
{
   'http': 'http://proxy.example.com:8080',
   'https': 'http://secureproxy.example.com:8090',
}

```
or
```
{
   'http': 'http://user:pass@1.1.1.1:8080',
   'https': 'http://user:pass@1.1.1.1:8090',
}
```
or
```
 {
    'http': 'http://45.95.147.106:8080',
    'https': 'https://37.187.17.89:3128'
}
```
https://github.com/masterking32/MasterHamsterKombatBot/blob/bb7010521b89a25e4d5a06174942a8ef95bc2fc9/main.py#L39

For Socks 

Run this: `pip install requests[socks]`

```
{
    'http': 'socks5://24.249.199.4:41458',
    'https': 'socks5://24.249.199.4:41458'
}
```
